exec/java-exec/src/main/java/org/apache/drill/exec/store/parquet/ParquetGroupScan.java

```diff
@@ -716,6 +726,7 @@ public String toString() {
     return "ParquetGroupScan [entries=" + entries
         + ", selectionRoot=" + selectionRoot
         + ", numFiles=" + getEntries().size()
+        + ", usedCache=" + usedMetadataCache
```

Comment:
We only use the cache if there is a cache file for the root directory of the query.

I can change the name to useMetadataFile